app/models/movie.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Movie < ApplicationRecord
+  has_many :screenings
```

Comment:
It might be a good idea to define [dependent](https://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many) option. It's good to mark it explicit even when using default `nil` option